docs-starlight/src/data/flags/dependency-fetch-output-from-state.mdx

```diff
@@ -16,7 +16,11 @@ The main benefit this flag provides is performance. Reading directly from state
 The limitation of this approach is that it is only supported by the S3 backend, and OpenTofu/Terraform may change the schema of the state file in the future, breaking this functionality.
 
 <Aside type="caution">
-This is an experimental feature. When enabled, Terragrunt will fetch dependency outputs directly from the state file instead of using `tofu output`.
+This is an experimental feature, use with caution.
```

Comment:
What does "use with caution" really mean?  What should people be aware of?  IMO we should be pretty explicit about when to use/not use based on tradeoffs.  Yes its ultimately end user responsibility to maintain their stuff, but they come to use to get best practice and to gain understanding.  We're the experts.

Reply:
It's the next line...

There is no guarantee that OpenTofu/Terraform will maintain the existing schema of their state files, so there is also no guarantee that the flag will work as expected in future versions of OpenTofu/Terraform. Also, note that the only backend that supports this feature is the AWS S3 backend.